src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/General/PreferencesType.php

```diff
@@ -90,42 +90,121 @@ public function buildForm(FormBuilderInterface $builder, array $options)
+                ),
+                'help' => $this->trans(
+                    'When enabled, all the pages of your shop will be SSL-secured.',
+                    'Admin.Shopparameters.Help'
+                ),
             ])
             ->add('enable_token', SwitchType::class, [
                 'disabled' => !$this->isContextDependantOptionEnabled(),
+                'label' => $this->trans(
+                    'Increase front office security',
+                    'Admin.Shopparameters.Feature'
+                ),
+                'help' => $this->trans(
+                    'Enable or disable token in the Front Office to improve PrestaShop\'s security.',
+                    'Admin.Shopparameters.Help'
+                ),
             ])
-            ->add('allow_html_iframes', SwitchType::class)
-            ->add('use_htmlpurifier', SwitchType::class)
-            ->add('price_round_mode', ChoiceType::class, [
-                'choices_as_values' => true,
-                'choices' => [
-                    'Round up away from zero, when it is half way there (recommended)' => $configuration->get('PS_ROUND_HALF_UP'),
-                    'Round down towards zero, when it is half way there' => $configuration->get('PS_ROUND_HALF_DOWN'),
-                    'Round towards the next even value' => $configuration->get('PS_ROUND_HALF_EVEN'),
-                    'Round towards the next odd value' => $configuration->get('PS_ROUND_HALF_ODD'),
-                    'Round up to the nearest value' => $configuration->get('PS_ROUND_UP'),
-                    'Round down to the nearest value' => $configuration->get('PS_ROUND_DOWN'),
-                ],
+            ->add('allow_html_iframes', SwitchType::class, [
+                'label' => $this->trans(
+                        'Allow iframes on HTML fields',
+                        'Admin.Shopparameters.Feature'
+                    ),
+                'help' => $this->trans(
+                    'Allow iframes on text fields like product description. We recommend that you leave this option disabled.',
+                    'Admin.Shopparameters.Help'
+                ),
+            ])
+            ->add('use_htmlpurifier', SwitchType::class, [
+                'label' => $this->trans(
+                        'Use HTMLPurifier Library',
+                        'Admin.Shopparameters.Feature'
+                    ),
+                'help' => $this->trans(
+                    'Clean the HTML content on text fields. We recommend that you leave this option enabled.',
+                    'Admin.Shopparameters.Help'
+                ),
             ])
+            ->add(
+                'price_round_mode', ChoiceType::class, [
+                    'placeholder' => false,
+                    'choices_as_values' => true,
+                    'choices' => [
+                        'Round up away from zero, when it is half way there (recommended)' => $configuration->get('PS_ROUND_HALF_UP'),
```

Comment:
One question about those choice fields They are not translated here But I believe Symfony has automatic translation for choices fields and you can even define a translation domain

So I guess we should use `Admin.Shopparameters.Feature` as `choice_translation_domain` option (considering these wordings are present in this domain)

Reply:
Ok so it seems it actually works because the template has this `{% trans_default_domain "Admin.Shopparameters.Feature" %}` So the correct domain will be used anyway, so it's not blocking but still maybe it would be better to explicitely set the choice domain in the form?
This is not blocking for the PR anyway and can be improved later, alongs with other similar cases I guess

Reply:
Now I think I remember Mickael saying something like "we cannot use some Symfony Form features like Constraints because Symfony expects only 1 domain" 🤔